prov/shm/src/smr_progress.c

```diff
@@ -1062,18 +1149,65 @@ static void smr_progress_cmd(struct smr_ep *ep)
 	}
 }
 
+static void smr_progress_ipc_list(struct smr_ep *ep)
```

Comment:
Merge with sar list progressing per first comment

Reply:
as discussed in the earlier comment, it makes more sense to keep this separate.